### PR TITLE
More things to es6 classes

### PR DESCRIPTION
**Pull request by jasongrout**

### CODE REVIEW

File: js/src/PanZoomModel.ts [hunk 1]
```diff
@@ -31,41 +31,41 @@ export const PanZoomModel = basemodel.BaseModel.extend({
...
-    },
+    }
 
-    reset_scales: function() {
-        var that = this;
-        widgets.resolvePromisesDict(this.get("scales")).then(function(scales: any) {
+    async reset_scales() {
+        widgets.resolvePromisesDict(this.get("scales")).then((scales: any) => {
             _.each(Object.keys(scales), function(k) {
                 _.each(scales[k], function(s: any, i) {
```
Comment (martinRenou) [inline js/src/PanZoomModel.ts]: Maybe you need to make them arrow functions?

Comment (jasongrout) [reply js/src/PanZoomModel.ts]: oops, I did halfway, but then aborted, but you're right that it's wrong now.

Comment (jasongrout) [reply js/src/PanZoomModel.ts]: Done!

**Comment by jasongrout** [comment]
@martinRenou - everything look okay now?

**Comment by martinRenou** [comment]
Yep!

**Pull request merged**